augeas-mode.el

```diff
@@ -74,7 +74,7 @@ Argument IGNORE is not used here."
 (defvar augeas-mode-syntax-table
   (let ((st (make-syntax-table)))
       (modify-syntax-entry ?_ "w"    st)
-      (modify-syntax-entry ?\" "_"   st)
+      (modify-syntax-entry ?\" "."   st)
```

Comment:
I see that this is wrong. I was trying to fix multi-line strings. After reverting just this change, the following example is broken:
```
let foo = "asdf
/ghjk/"
```

The quote characters have no special face, `asdf` has `font-lock-variable-name-face`, and `/ghjk/` has `font-lock-string-face`.

I believe that relying on the syntax table at all for strings and regular expressions won't work...